### ISSUE TITLE
Build failure on project asqatasun-engine with openJDK-8

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Asqatasun#115]
## Observed behaviour
"Failed to execute goal maven-surefire-plugin" on project **asqatasun-engine**

discussion in French forum
http://forum.asqatasun.org/t/pb-de-compilation-suite-a-migration-os-ubuntu-14-04-remplace-par-16-04/164

## configuration 
2 configurations for the same result

### Ubuntu 16.04 Desktop
* Ubuntu 16.04
* openjdk-8-jdk 
```
git clone (...)
cd Asqatasun
mvn clean install
```

## Docker maven:3-jdk-8
```
git clone (...)
cd Asqatasun
docker pull maven:3-jdk-8
docker run -it --rm --name mvn-jdk8 -v "$PWD":/usr/src/asqa -w /usr/src/asqa maven:3-jdk-8 mvn clean install
```

**Comment by dzc34** [comment]
7862d29249249a9e968c0ec3e445678af91bf592 fix BUILD FAILURE "Failed to execute goal maven-surefire-plugin" on project asqatasun-engine:

**Comment by dzc34** [comment]
fixed 333288e05f54b5f2a16d82f4edefea1878270ed3 "Fix Java 8 build + issue #113"
